lua/lvim/core/indentlines.lua

```diff
@@ -20,8 +20,8 @@ M.config = function()
       char = lvim.icons.ui.LineLeft,
       show_trailing_blankline_indent = false,
       show_first_indent_level = true,
-      use_treesitter = false,
-      show_current_context = false,
+      use_treesitter = true,
+      show_current_context = true,
```

Comment:
is this originally tied to tree-sitter?

Reply:
it was true, but we changed that when implementing the first big files handling commit

Reply:
it does depend on tree-sitter, not sure about the performance implications tho

```vimhelp
g:indent_blankline_show_current_context *g:indent_blankline_show_current_context*

    When on, use treesitter to determine the current context. Then show the
    indent character in a different highlight.

    Note: Requires https://github.com/nvim-treesitter/nvim-treesitter to be
          installed

    Note: With this option enabled, the plugin refreshes on |CursorMoved|,
          which might be slower

    Default: v:false                                                          

    Example:  

        let g:indent_blankline_show_current_context = v:true


```

Reply:
it got changed to false here https://github.com/LunarVim/LunarVim/pull/3268/files#diff-458a835250c905af56d022a4a9ec0180a864c15ca07e5a3c190965538086711dL23

Reply:
as for performance: bigfile.nvim disables indent_blankline